### ISSUE TITLE
Use switchboard to run Insights surveys

### ISSUE THREAD

**Issue by bbinto** [mozilla-mobile/focus-android#1430]
UX #1511 

Blocked on  #1030.

Invite existing users to participate in survey

Goal:
- To gain insights about our existing Focus user, and to understand what features they consider important to, missing from Focus
- To gain insights about use cases when our users pick up Focus
- To understand what kind of users are most valuable to us (retain the best)
- Does convenience win over privacy?

More: https://docs.google.com/document/d/1lLG-DHLI9WZW2oRszjL7i0wVl96P06Y81lx1WjvKspM/edit#heading=h.nldib6sb6iko

**Comment by pocmo** [comment]
Depends on #2350

**Comment by bbinto** [comment]
closing in favour of https://github.com/mozilla-mobile/focus-android/issues/1431
